sklearn/tests/test_common.py

```diff
@@ -17,6 +17,7 @@
 
 import pytest
 
+from scipy.linalg import LinAlgWarning
```

Comment:
Could you move this import in the block above of the third-party libraries import